src/components/learner-credit-management/data/utils.js

```diff
@@ -41,19 +41,24 @@ export const transformOfferSummary = (offerSummary) => {
     percentUtilized,
   };
 };
+function getRandom(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
 
 /**
  * Transforms enrollment data from analytics api to fields for display
  * in learner credit allocation table.
+ * Any non-existent enterpriseEnrollmentId will be assigned a 12 digit random uuid number.
+ * Any non-existent userEmail will be assigned as an empty string.
  *
  * @param {array} results List of raw enrollment results from API.
  *
  * @returns List of transformed results for display in table.
  */
 export const transformUtilizationTableResults = results => results.map(result => ({
   created: result.created,
-  enterpriseEnrollmentId: result.enterpriseEnrollmentId,
-  userEmail: result.userEmail,
+  enterpriseEnrollmentId: result.enterpriseEnrollmentId === null ? parseInt(uuidv4().replace(/[^\d]+/g, getRandom(0, 9)).slice(0, 12), 10) : result.enterpriseEnrollmentId,
```

Comment:
Could this be written as `result.enterpriseEnrollmentId || uuidv4()`? I guess I'm not exactly following why we'd need `getRandom` if we're using `uuidv4`?